extensions/adobe/experience/journeyOrchestration/stepEvents/journeyStepEventJourneyFieldsMixin.schema.json

```diff
@@ -56,6 +56,13 @@
               "type": "string",
               "meta:titleId": "journeyFields##https://ns.adobe.com/experience/journeyOrchestration/version##title##96811",
               "meta:descriptionId": "journeyFields##https://ns.adobe.com/experience/journeyOrchestration/version##description##49381"
+            },
+            "https://ns.adobe.com/experience/journeyOrchestration/objectiveMetricId": {
+              "title": "objectiveMetricId",
+              "description": "Id of the objective metric.",
+              "type": "string",
+              "meta:titleId": "journeyFields##https://ns.adobe.com/experience/journeyOrchestration/objectiveMetricId##title##55013",
```

Comment:
Please remove `"meta:titleId` and `meta:descriptionId` as they are auto-generated fields